README.md

```diff
@@ -123,7 +123,13 @@ The following table lists all valid connection properties:
 | PRIVATE_KEY_PWD            |No        | The passphrase to use for decrypting the private key, if the key is encrypted.|
 | PRIVATE_KEY                |Depends   | The private key to use for key-pair authentication. Must be used in combination with AUTHENTICATOR=snowflake_jwt. <br /> If the private key value includes any equal signs (=), make sure to replace each equal sign with two signs (==) to ensure that the connection string is parsed correctly.|
 | TOKEN                      |Depends   | The OAuth token to use for OAuth authentication. Must be used in combination with AUTHENTICATOR=oauth.|
-| INSECUREMODE               |No   	    | Set to true to disable the certificate revocation list check. Default is false.|
+| INSECUREMODE               |No   	| Set to true to disable the certificate revocation list check. Default is false.|
+| USEPROXY                   | No       | Set to true if you need to use a proxy server. The default value is false. |
+| PROXYHOST                  | No       | The hostname of the proxy server. |
+| PROXYPORT                  | No       | The port number of the proxy server. |
+| PROXYUSER                  | No       | The username for authenticating to the proxy server. |
+| PROXYPASSWORD              | No       | The password for authenticating to the proxy server. |
+| NONPROXYHOSTS              | No       | The list of hosts that the driver should connect to directly, bypassing the proxy server. Separate the hostnames with a URL-escaped pipe symbol (`%7C`). |
```

Comment:
Is it correct that the pipe symbol should be URL-escaped?

Reply:
Does this support using an asterisk (`*`) as a wildcard?

Reply:
So, for '%7c', I did this based on the implementation for JDBC. But it is not necessary, if the user passes a '|' character, I've tested it and it's Ok (same for JDBC as well).

Yes, the wildcard * is supported

Reply:
Thanks! I changed this to use a `|` symbol and mentioned the asterisk as well.